netty/src/main/java/io/grpc/netty/NettySocketSupport.java

```diff
@@ -20,14 +20,15 @@
 import com.google.common.collect.ImmutableMap;
 import io.grpc.internal.Channelz.TcpInfo;
 import io.netty.channel.Channel;
-import java.util.HashMap;
 import java.util.Map;
+import java.util.logging.Logger;
 import javax.annotation.Nullable;
 
 /**
  * An class for getting low level socket info.
  */
 final class NettySocketSupport {
+  private static final Logger log = Logger.getLogger(NettySocketSupport.class.getName());
```

Comment:
???

Reply:
This will be used in the internal version, I suppose it's ok for it not to be present at all in the common version. I was trying to minimize the number of text replacements in this file.